src/server/auth/arboristClient.js

```diff
@@ -65,6 +65,64 @@ class ArboristClient {
       },
     );
   }
+
+  listAuthorizedCreateResources(jwt) {
+    // Make request to arborist for list of resources with access
+    const resourcesEndpoint = `${this.baseEndpoint}/auth/mapping`;
+    log.debug('[ArboristClient] listAuthorizedCreateResources jwt: ', jwt);
+
+    const headers = (jwt) ? { Authorization: `bearer ${jwt}` } : {};
+    return fetch(
+      resourcesEndpoint,
+      {
+        method: 'POST',
+        headers,
+      },
+    ).then(
+      (response) => {
+        if (response.status === 400) {
+          // Retry with GET instead of POST. Older version of Arborist POST auth/mapping
+          // didn't support token authentication.
+          // This catch block can be removed in a little while, when it will likely not cause issues
+          return fetch(
+            resourcesEndpoint,
+            {
+              method: 'GET',
+              headers,
+            },
+          ).then((res) => res.json());
+        }
+        return response.json();
+      },
+      (err) => {
+        log.error(err);
+        throw new CodedError(500, err);
+      },
+    ).then(
+      (result) => {
+        const data = {
+          resources: [],
+        };
+        Object.keys(result).forEach((key) => {
+        // logic: you have access to a project if you have the following access:
+        // method 'create' (or '*' - all methods) to service 'guppy' (or '*' - all services)
+        // on the project resource.
```

Comment:
This is too broad, and it sort of binds the two roles of data submitter and Guppy admin togather. The restarting of Guppy should be an "admin-level" feature. I think we'd better to have a dedicated "guppy_admin" policy for this feature, which could be `service == guppy, method == *`, or `service == guppy, method == admin_access`

Reply:
see https://github.com/uc-cdis/guppy/pull/273/commits/713fc1fac7a91b471b6c8696843e7c9d23a407d9